app/cypress/e2e/SecvisogramPage/FormEditorTab.cy.js

```diff
@@ -607,4 +607,202 @@ describe('SecvisogramPage / FormEditor Tab', function () {
+      cy.visit('?tab=EDITOR')
+
+      // input should be empty first
+      cy.get('[data-testid="menu_entry-/document/tracking"]').click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="date"]'
+      ).should('be.empty')
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="time"]'
+      ).should('be.empty')
+
+      // generate button should do nothing without revision history entries
+      cy.get(
+        '[data-testid="document-tracking-current_release_date-generateButton"]'
+      )
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="date"]'
+      ).should('be.empty')
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="time"]'
+      ).should('be.empty')
+
+      // create new revision history item
+      cy.get(
+        '[data-testid="menu_entry-/document/tracking/revision_history-add_item_button"]'
+      ).click({ force: true })
+      cy.get(
+        '[data-testid="menu_entry-/document/tracking/revision_history/0"]'
+      ).click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-revision_history-0-date"] input[type="date"]'
+      )
+        .invoke('attr', 'value', '2020-01-01')
+        .trigger('change')
+      cy.get(
+        '[data-testid="attribute-document-tracking-revision_history-0-date"] input[type="time"]'
+      )
+        .invoke('attr', 'value', '12:00')
+        .trigger('change')
+
+      // current release should still be empty
+      cy.get('[data-testid="document/tracking-fieldButton"]').click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="date"]'
+      ).should('be.empty')
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="time"]'
+      ).should('be.empty')
+
+      // generate button should enter correct date and time
+      cy.get(
+        '[data-testid="document-tracking-current_release_date-generateButton"]'
+      ).click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="date"]'
+      ).should('have.value', '2020-01-01')
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="time"]'
+      ).should('have.value', '12:00')
+
+      // just regenerating should be still the same
+      cy.get(
+        '[data-testid="document-tracking-current_release_date-generateButton"]'
+      ).click()
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="date"]'
+      ).should('have.value', '2020-01-01')
+      cy.get(
+        '[data-testid="attribute-document-tracking-current_release_date"] input[type="time"]'
+      ).should('have.value', '12:00')
+    })
+
+    it('fill initial release date', function () {
+      cy.visit('?tab=EDITOR')
```

Comment:
This test fails for me locally.

Reply:
both tests work for me locally and on github.
Could you comment your error message?

Reply:
I changed the way date and time values are invoked, maybe that fixed the issue for you

Reply:
Works now. Thanks.